src/main/java/commands/Command.java

```diff
@@ -0,0 +1,29 @@
+package commands;
+
+import core.Ui;
+import data.Storage;
+import tasks.TaskList;
+
+/**
```

Comment:
I like that there is a descriptive header comment for the public class, which explains the class very clearly.

Reply:
Thank you!